ios/ble/peripheral/PeripheralCommunicatorDelegate.swift

```diff
@@ -0,0 +1,13 @@
+
+import Foundation
+
+protocol PeripheralCommunicatorDelegate: AnyObject {
+    func transmissionReportHandler(data: Data?)
+    func writeSuccessHandler()
+    func verificationStatusChange(data: Data?)
+}
+
+protocol WalletProtocol: AnyObject {
```

Comment:
One is called protocol and the other is named delegate, stick to one pattern here

Reply:
Using protocols here